docs/gettingstarted/query_test_report.rst

```diff
@@ -45,6 +45,34 @@ You may find it useful to fetch all failed records and determine pass/fail crite
 
 .. _report_format_fields:
 
+Detailed Reports (``buildtest report --detailed``)
+---------------------------------------------------
+
+The **buildtest report --detailed** option can be used to show a detailed test report that may be of interest when
+examining a test. This option is synonymous to running ``buildtest report --format name,id,user,state,returncode,runtime,outfile,errfile,buildspec``.
+Shown below is an example output.
+
+.. dropdown:: ``buildtest report --detailed``
+
+    .. command-output:: buildtest report --detailed
+
+Notes
+*******
+
+.. note::
+The detailed and --format options are mutually exclusive options because both options will alter the format fields when displaying test results.
+Later in text after the note state the following.
+
+.. code-block:: RST
```

Comment:
This code block is not right you should remove this and refer to sphinx documentation bcz you don't have a clear understanding of what this is used for. You don't code format RST rather you format code that is console, bash, etc.